### DISCUSSION TITLE
css is not a function

### DISCUSSION THREAD

**Discussion by DashaFah** [gohugo-ananke/ananke#722] [2 - Q&A / Support] [answered]
Hello,
the file "layouts/partials/func/style/GetMainCSS.html" is broken and returns an error "css is not a function" when deploying. 

Solution: I changed css.Sass back to resources.ToCSS and fixed my issue

**Comment by davidsneighbour** [comment, marked as answer]
This function is available from Hugo 0.128.+ on. Please update your installation to use the new features.

**Reply by miaucl** [reply]
The build on netlify complains about the same problem.

**Reply by davidsneighbour** [reply]
Same solution here ;) You need to setup the proper Hugo version in your `netlify.toml`. Something like this:

```toml
[context.production.environment]
HUGO_VERSION = "0.133.1"
```

Use the latest version and update regularly when a new Hugo version is released.

**Reply by miaucl** [reply]
I have never used netlify before and it is not set up in my github fork. Will have to see how to do that as I am a newbie there :)

**Reply by davidsneighbour** [reply]
What do you mean by "the build on netlify" then? The sample page? I have no access there yet. Will try to find out what to do with it.

**Reply by miaucl** [reply]
Talking about this comment: https://github.com/theNewDynamic/gohugo-theme-ananke/pull/732#issuecomment-2408553496

And when I click on "Why did it fail?"

<img width="1512" alt="Bildschirmfoto 2024-10-12 um 15 38 45" src="https://github.com/user-attachments/assets/cc02cead-d007-4b3d-a2a9-75a7dffeaa0c">
